README.md

```diff
@@ -208,3 +208,17 @@ You should probably increase DMA coherent pool using `coherent_pool=32M` kernel
 [ 7.171632] xtrx 0000:01:00.0: Failed to register TX DMA buffers.
 ```
 For the details see issue [#37](https://github.com/xtrx-sdr/images/issues/37).
+
+## How do I test GPS and specifically PPS?
+
+For GPS data like current time and geoposition:
+1. Connect your antenna.
+2. Install XTRX driver and libraries (use [pre-built packages](https://github.com/xtrx-sdr/images/blob/master/README.md#pre-built-packages) or build them from sources).
```

Comment:
This is not a part of the testing steps... Mention before the steps list that the prerequisite is to (1) connect XTRX with miniPCIe or PCIe because USB3 libs do not expose GPS to the system, and (2) make sure XTRX PCIe drivers are installed.
You don't need XTRX libraries since it solely depends on kernel devices and `gpsd` software.